### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by Simpleshaikh1**
I have added styling to my project to replicate the design provided

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -13,10 +13,12 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
 }
 
 .book-container {
   display: flex;
```
Comment (1GORDON) [inline src/App.css:20]: Great job @Simpleshaikh1 🤝

- [ ] As stated in the project requirements, kindly ensure you do not repeat the same rule. You may create a single class for the `display: flex`, and the `align-items: center` rules, and then reuse these rules in your JSX code. Kindly use this [article](https://speckyboy.com/good-bad-css-practices/) for more guidance.

File: src/components/Book.jsx [hunk 2]
```diff
@@ -36,14 +42,18 @@ const Book = ({ book }) => {
 
       <div className="progressBar">
         <CircularProgressbar
-          value={parseInt(book.completed, 30)}
-          text={`${book.completed}`}
+          value={parseInt(boook.completed, 30)}
+          // text={`${boook.completed}`}
```
Comment (1GORDON) [inline src/components/Book.jsx]: Great job @Simpleshaikh1 🤝

- [ ] Kindly remove commented code from your project as this is regarded as bad practice.

File: src/App.css [hunk 3]
```diff
@@ -13,10 +13,12 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
 }
```
Comment (1GORDON) [inline src/App.css:18]: Great job @Simpleshaikh1 🤝

- [ ] As given in the project requirements, we are supposed to adhere to the design given in order to create a replica of this [given design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png).
- [ ] Kindly ensure the borders are removed from the `progress buttons`, and the `add book` button.
- [ ] Also ensure the font colors of the texts and the colors of the buttons on your design match the design provided.
- [ ] Kindly ensure the elements of your design are well aligned just like in the design provided.
your-cms

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Simpleshaikh1 ✋

Good job so far!

## Highlights!
- Basic styling well used ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Simpleshaikh1,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing changes as requested!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**